templates/sudoers.aix.erb

```diff
@@ -83,11 +83,13 @@ Defaults!<%= command %> !log_output
 ##
 root ALL=(ALL) ALL
 
+<% if @wheel_config == 'password' %>
 ## Uncomment to allow members of group wheel to execute any command
-# %wheel ALL=(ALL) ALL
```

Comment:
I'm not quite sure if it makes sense to remove this line, but leave the comment above.
I'd rather leave the commented version, but I can see, that this might be a bit of work.

Reply:
Oh forget about it. Was missreading it :)